usaspending_api/search/delta_models/award_search.py

```diff
@@ -76,7 +76,7 @@
     "pop_congressional_population": {"delta": "INTEGER", "postgres": "INTEGER"},
     "cfda_program_title": {"delta": "STRING", "postgres": "TEXT"},
     "cfda_number": {"delta": "STRING", "postgres": "TEXT"},
-    "cfdas": {"delta": "ARRAY<STRING>", "postgres": "[TEXT]"},
+    "cfdas": {"delta": "ARRAY<STRING>", "postgres": "[TEXT]"},  # TODO: <---- these pg text array types valid? = TEXT[]
```

Comment:
Yeah transaction_search seems to have it as `TEXT[]`. I thought I saw `[TEXT]` as valid and working but regardless, this is a good note to check/test soon.

Reply:
I think I might just change it in this PR.